### PR TITLE
fix: block resize with long content and broken image path

### PR DESCRIPTION
**Pull request by ihor-romaniuk**
### Description
This pull request contains:
- styling fixes of Problem content editor for correct displaying with long/wide content with adding block limits and horizontal scroll
- fix broken images URL in hints and answers blocks

#### Screenshots before:
| image | image |
|---|---|

#### Screenshots after:
| image | image |
|---|---|

#### Steps to Reproduce: 
1. Enable new problem editor and sharing by adding in `/admin/waffle/flag/`
- [new_core_editors.use_new_problem_editor](http://localhost:18000/admin/waffle/flag/9/change/)
2. In studio open unit -> add new component -> Problem -> Single / Multiple select
3. Add images, table to hints and answers blocks
4. Check problem displaying

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @ihor-romaniuk! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

File: src/editors/data/redux/thunkActions/problem.js [hunk 1]
```diff
@@ -47,15 +48,17 @@ export const getDataFromOlx = ({ rawOLX, rawSettings, defaultSettings }) => {
...
 };
 
-export const loadProblem = ({ rawOLX, rawSettings, defaultSettings }) => (dispatch) => {
+export const loadProblem = ({ rawOLX, rawSettings, defaultSettings }) => (dispatch, getState) => {
   if (isBlankProblem({ rawOLX })) {
     dispatch(actions.problem.setEnableTypeSelection(camelizeKeys(defaultSettings)));
   } else {
-    dispatch(actions.problem.load(getDataFromOlx({ rawOLX, rawSettings, defaultSettings })));
+    const dataFromOlx = getDataFromOlx({ rawOLX, rawSettings, defaultSettings });
+    preprocessProblemData(dataFromOlx, getState());
```
Comment (KristinAoki) [inline src/editors/data/redux/thunkActions/problem.js:56]: This should be happening when the tinyMCE editor loads, so it is not necessary to preprocess the problem here. See https://github.com/openedx/frontend-lib-content-components/blob/main/src/editors/sharedComponents/TinyMceWidget/hooks.js#L74-L97

Comment (KristinAoki) [reply src/editors/data/redux/thunkActions/problem.js:56]: If this is not longer needed, please remove all changes related to `preprocessProblemData`

File: src/editors/data/services/cms/utils.js [hunk 2]
```diff
@@ -22,3 +22,19 @@ export const post = (...args) => getAuthenticatedHttpClient().post(...args);
...
+ *
+ * @param {string} text - The input text containing image tags with relative URLs.
+ * @param {string} lmsEndpointUrl - The base URL to prepend to each relative URL to form an absolute URL.
+ * @returns {string} The text with all relative image URLs replaced by absolute URLs.
+ */
+export const replaceRelativeImageUrlsByAbsolute = (text, lmsEndpointUrl) => {
+  const relativeSrcRegExp = /(?<=src=")(\/.*?)(?=")/g;
+  const replaceRelativeToAbsoluteUrl = (relativeUrl) => `${lmsEndpointUrl}${relativeUrl}`;
+  return (text || '').replaceAll(relativeSrcRegExp, replaceRelativeToAbsoluteUrl);
+};
```
Comment (KristinAoki) [inline src/editors/data/services/cms/utils.js:40]: See comment regarding `preprocessProblemData` and remove if necessary.

**Review by KristinAoki — CHANGES_REQUESTED** [review]

**Comment by KristinAoki** [comment]
Hi! Thank you for your contribution to this repo. This repo is being deprecated and all the code is being moved to `frontend-app-course-authoring`. If you would like your work to be included in the transition please resolve outstanding requests for this PR. This PR will automatically close once `frontend-app-course-authoring` PR [#1208](https://github.com/openedx/frontend-app-course-authoring/pull/1208) is merged.

**Comment by bradenmacdonald** [comment]
Closing as we've now moved this code, per the notice above. Please re-open this PR against https://github.com/openedx/frontend-app-course-authoring/ if you'd like to continue it.

**Comment by openedx-webhooks** [comment]
@ihor-romaniuk Even though your pull request wasn’t merged, please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-lib-content-components&entry.867055334=https://github.com/openedx/frontend-lib-content-components/pull/478&entry.1484655318=https://github.com/ihor-romaniuk&entry.752974735=2024-05-29+10:45&entry.1917517419=2024-08-29+19:05&entry.2133058324=No) so we can improve your experience in the future.

**Pull request closed without merging**